src/components/GameView/GameOverDialog.vue

```diff
@@ -1,61 +1,61 @@
 <template>
-  <v-dialog v-model="show" persistent max-width="650">
-    <v-card id="game-over-dialog">
-      <div id="dialog-header">
-        <div v-if="currentMatch" class="d-flex">
-          <v-card-title :data-cy="headingDataAttr" class="mt-8">
-            <h1 class="dialog-header">{{ heading }}</h1>
-          </v-card-title>
-          <v-card-text class="d-flex justify-end mr-4 mt-4">
-            <v-img :src="logoSrc" :data-cy="logoDataAttr" class="logo-image-match" />
-          </v-card-text>
-        </div>
-        <div v-else>
-          <v-card-title :data-cy="headingDataAttr">
-            <h1 class="dialog-header">{{ heading }}</h1>
-          </v-card-title>
-          <v-card-text class="d-flex justify-center">
-            <v-img :src="logoSrc" :data-cy="logoDataAttr" class="logo-image" />
-          </v-card-text>
+  <base-dialog v-model="show" id="game-over-dialog">
+    <template #title>
+      <h1 :data-cy="headingDataAttr" class="dialog-header">{{ heading }}</h1>
+      <v-img v-if="currentMatch && !isMobilePortrait" :src="logoSrc" :data-cy="logoDataAttr" class="logo-image-match" />
+    </template>
+
+    <template #body>
+      <template v-if="isMobilePortrait || !currentMatch">
+        <div class="d-flex justify-center">
+          <v-img :src="logoSrc" :data-cy="logoDataAttr" class="logo-image" />
```

Comment:
The centered image now appears on mobile portrait for ranked games in addition to showing up for unranked games on all device sizes.

Mobile portrait for ranked game
![image](https://user-images.githubusercontent.com/6520704/233677649-b93d7198-902d-466e-bb7a-25c8a596baef.png)

Midsized non-ranked
![image](https://user-images.githubusercontent.com/6520704/233677907-2267aaf5-4a70-47df-a9cb-b969a333b4f9.png)